core/domain/user_jobs_continuous.py

```diff
@@ -249,7 +252,8 @@ def map(item):
                 feconf.COMMIT_MESSAGE_EXPLORATION_DELETED))
 
         for exp_model in tracked_exp_models_for_feedback:
-            threads = feedback_services.get_all_threads(exp_model.id, False)
+            threads = feedback_services.get_all_threads(
+                feconf.ENABLE_GENERALIZED_FEEDBACK_THREADS, exp_model.id, False)
```

Comment:
The first value here seems weird. If it is wrong, how did this pass tests? Write a test (or add checks) that guard against this error and make sure they fail before you change this value to the correct one.

Reply:
Oops! What happened is in the tests, I forgot to swap the ENABLE_GENERALIZED_FEEDBACK_THREADS variable to true. In the old framework, we don't use the first parameter in the function call. On switching the flag in the test suite (i.e. while using the new framework, which uses the first parameter), the test failed!

Reply:
Fixed. Also there were some fixes required to actually make it pass in the new system. Now it works fine!